src/FileManipulator.php

```diff
@@ -109,7 +114,7 @@ public function createTempDirectory() : string
         return $tempDirectory;
     }
 
-    protected function convertToImage(string $pdfFile) : string
+    protected function convertPDFToImage(string $pdfFile) : string
```

Comment:
Nitpick: I don't like having extensions as capitals in function names. Could you make the function name camel case?